sql/sqlserver/server.go

```diff
@@ -563,25 +561,51 @@ func (s *Server) Select(session *Session, p *parser.Select, args []sqlwire.Datum
 	return nil
 }
 
+// Set sets session variables.
+func (s *Server) Set(session *Session, p *parser2.Set, args []sqlwire.Datum,
+	resp *sqlwire.Response) error {
+	// By using QualifiedName.String() here any variables that are keywords will
+	// be double quoted.
```

Comment:
`QualifiedName` is a dot separated list of identifiers. Double quotes are used for identifiers that are also keywords (such as "database"). Single quotes are used for strings.